OrbitGl/EventTrack.cpp

```diff
@@ -208,7 +208,7 @@ std::string EventTrack::GetSampleTooltip(PickingId id) const {
   }
 
   const CallstackData* callstack_data = GOrbitApp->GetCaptureData().GetCallstackData();
-  CallstackEvent* callstack_event = static_cast<CallstackEvent*>(user_data->custom_data_);
+  const auto callstack_event = static_cast<const CallstackEvent*>(user_data->custom_data_);
```

Comment:
I don't know if we have general guidance here, but I would use `const auto*` here, `auto` because of the `static_cast` but still making clear it's a pointer. I think it's done like this in many other places when casting pointers.

Reply:
yep, think this is better. thanks